scripts/zones/Ceizak_Battlegrounds/mobs/Transcendent_Scorpion.lua

```diff
@@ -0,0 +1,62 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Transcendent Scorpion
+-----------------------------------
```

Comment:
This will also need comments with commands for `!pos` (take you to the NM's spawn point), and `!additem` or some sort of `!exec` to force-pop the NM, for easier testing